packages/paste-core/layout/stack/__tests__/stack.test.tsx

```diff
@@ -112,7 +120,22 @@ describe('Stack', () => {
   });
 
   it('should render as a header', () => {
-    const {asFragment} = render(<MockHeaderStack />);
-    expect(asFragment()).toMatchSnapshot();
+    const {getByTestId} = render(<MockHeaderStack />);
+    expect(getByTestId('header')).not.toBeNull();
```

Comment:
You need to test the element tag name or something here. This only tests that it exists by test id. If I change the as prop, the test will still pass

Reply:
Changed that to test for the actual element.